khelo/lib/ui/flow/profile/profile_view_model.dart

```diff
@@ -25,6 +28,9 @@ final profileStateProvider =
 });
 
 class ProfileViewNotifier extends StateNotifier<ProfileState> {
+  static const String iosAppId = '6480175424';
```

Comment:
If you don't need to use the appId in multiple places, define it directly in the URL instead of creating a separate variable.